index.html

```diff
@@ -0,0 +1,1048 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=0.25, user-scalable=no"
+    />
+    <meta
+      name="description"
+      content="The homepage of Pixmart, that represents main features of the platform"
+    />
+    <title>Pixmart</title>
+    <link
+      rel="apple-touch-icon"
+      type="image/svg+xml"
+      href="./images/icons/favicon.svg"
+    />
+    <link rel="icon" type="image/svg+xml" href="./images/icons/favicon.svg" />
+    <link rel="alternate icon" type="image/png" href="./images/favicon.png" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Libre+Franklin:wght@400;500;600;700&display=swap"
+      rel="stylesheet"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Plus+Jakarta+Sans:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="./public/normalize.css" />
+    <link rel="stylesheet" href="./public/constants.css" />
+    <link rel="stylesheet" href="./public/animate.css" />
+    <link rel="stylesheet" href="./public/style.css" />
+  </head>
+  <body>
+    <header class="header list">
+      <a href="./index.html" class="header__logo">
```

Comment:
H1 tag has no descriptive text inside, but contains Logo image. Logo image in A tag is OK, but you need to add H1 tag with text separately. I would suggest you using Visually hidden technique for H1 as you do on Catalog page.

Reply:
2.8. ❤️ No gross markup errors.

Reply:
Fixed 🥇